### PR TITLE
Codeforces Questions

### PR DESCRIPTION
**Pull request by Isha-30**

### CODE REVIEW

File: Arrays/Reversing-Encryption.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import java.util.*;
+
+public class Main {
+    private static Scanner sc = new Scanner(System.in);
+
+    public static void main(String args[]) {
```
Comment (Nexengineer) [inline Arrays/Reversing-Encryption.java]: Please state the Questions URL or full question on top of the file

Comment (Nexengineer) [reply Arrays/Reversing-Encryption.java]: Please follow camelcase in file name

Comment (Isha-30) [reply Arrays/Reversing-Encryption.java]: Pasted the problem url on the top of the file
Name changed to CamelCase

**Comment by Nexengineer** [comment]
thanks

**Pull request merged**